### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mabablue**
added Mermaid diagrams and simplified the instructions

### CODE REVIEW

**Comment by mabablue** [comment]
can you please also change the following: editor into curator in the whole test; 
also we need to change the link to the nanopublication template for approving the metadata: https://nanodash.petapico.org/publish?34&template=https://w3id.org/np/RAi7_UxEF3TTPBp7lmWOVvKR-jUmZgDfY_ZIe57dqbOnQ&template-version=latest

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,74 +28,104 @@ flowchart TD
...
+As the editor perform the actions based on the reviewer decision. 
+- if the proposal is accepted with changes: negotiate with the reviewer and makes adjustments until both of you are satisfied;
+- if the reviewer accepts the proposal: proceed with the proposed action:
+
+3A. proposal - accept as is: approve the FSR nanopub with the [approval nanopub template](https://nanodash.knowledgepixels.com/publish?55&template=https://w3id.org/np/RApEyTXPOt_h81Ao0WpzRhigcgvqrbnNBCo8fEpsZ6CxU)  
+3B. proposal - accept with improvements: proceed with the improvement (see below)  
+3C. proposal - reject: the editor disapprove/retract the nanopub (see below).  
+
+
+3D. If the reviewer rejects your proposal, assign the issue to a second reviewer. You should follow the decision that has at least two supportersd.
```
Comment (rbramley) [inline CONTRIBUTING.md:126]: There's a typo at the end of this sentence "supportersd"

Comment (rbramley) [reply CONTRIBUTING.md:126]: I'll fix this now

**Comment by rbramley** [comment]
> can you please also change the following: editor into curator in the whole test; also we need to change the link to the nanopublication template for approving the metadata: https://nanodash.petapico.org/publish?34&template=https://w3id.org/np/RAi7_UxEF3TTPBp7lmWOVvKR-jUmZgDfY_ZIe57dqbOnQ&template-version=latest

Will address these now

**Comment by rbramley** [comment]
@mabablue would you like to preview my further changes before I merge?

File: CONTRIBUTING.md (hunk 1 above)
Comment (mabablue) [reply CONTRIBUTING.md]: thanks

**Review by mabablue — COMMENTED** [review]
looks good to me

**Review by rbramley — APPROVED** [review]

**Pull request merged**